s/primdata.ss

```diff
@@ -1831,12 +1831,14 @@
   (utf-16-codec [sig [() -> (codec)] [(sub-symbol) -> (codec)]] [flags pure true]) ; has optional eness argument
   (utf-16le-codec [sig [() -> (codec)]] [flags pure unrestricted true])
   (utf-16be-codec [sig [() -> (codec)]] [flags pure unrestricted true])
+  (vector-append [sig [(vector ...) -> (vector)]] [flags alloc safeongoodargs])
   (vector-cas! [sig [(nonempty-vector sub-index ptr ptr) -> (boolean)]] [flags cptypes2])
-  (vector-copy [sig [(vector) -> (vector)]] [flags alloc safeongoodargs])
+  (vector-copy [sig [(vector) -> (vector)] [(vector sub-index sub-index) -> (vector)]] [flags alloc])
```

Comment:
Should `(vector sub-index sub-index)` be `(vector sub-index sub-length)`?

Reply:
Looks like they should both be `sub-length`, since the first argument is allowed to match the length of the vector.